pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[project]
+name = "lan_pipe"
+version = "0.1.0"
+description = "LAN pipeline project"
+requires-python = ">=3.10,<3.11"
+dependencies = [
+    "jax[cuda11_pip]",
```

Comment:
The jax dependency is tricky, because it's setup dependent. 
Is there something smarter we could do here?